### DISCUSSION TITLE
More models into one scene

### DISCUSSION THREAD

**Discussion by phovorka** [google/model-viewer#3310] [Q&A]
Is possible to add more GLB models into one viewer (scene) and set up their position? This would spark many new implementation of product configurator.

**Comment by elalish** [comment]
Not yet, but there's lots of interest. We'd be interested in any details of use cases and potential APIs so that we can build something that will suite everyone's needs.

**Reply by slightHYP** [reply]
Just few simple considerations but they could be of interest.

The possibility to load separated 3D models in the same scene would allow to load only one single main 3D object (that will keep always the same geometries) attaching to it smaller 3D objects that rappresents particular details or customizations of the product.

The additional objects would be developed without touching the main object, the modelling phase could be externalize to another team and the 3d models could be served through dedicated API only when requested.

In this way the size of main GLTF could be smaller and optimize the loading performance.
